usecases/auth/authorization/types.go

```diff
@@ -60,10 +61,11 @@ var (
 	// Note:  if a new action added, don't forget to add it to availableWeaviateActions
 	// to be added to built in roles
 	// any action has to contain of `{verb}_{domain}` verb: CREATE, READ, UPDATE, DELETE domain: roles, users, cluster, schema, data
-	ManageRoles   = "manage_roles"
-	ReadRoles     = "read_roles"
-	ManageUsers   = "manage_users"
-	ManageCluster = "manage_cluster"
+	ManageRoles = "manage_roles"
+	ReadRoles   = "read_roles"
+	ManageUsers = "manage_users"
+	ReadCluster = "read_cluster"
+	ReadNodes   = "read_nodes"
```

Comment:
**issue (blocking):**  could you add it to availableWeaviateActions, so that it's automatically added to builtin Roles .
see [L61](https://github.com/weaviate/weaviate/pull/6467/files#diff-a1956a4e8b03f18a14b0a096631fb6d1a84e2b0e6a3ec54cf415e5e0186d2d44R61)

Reply:
I [did](https://github.com/weaviate/weaviate/pull/6467/files#diff-a1956a4e8b03f18a14b0a096631fb6d1a84e2b0e6a3ec54cf415e5e0186d2d44R97) 😁 Unless it's something else?

Reply:
🤦🏼‍♂️  didn't see it in the diff